lib/mastodon/accounts_cli.rb

```diff
@@ -397,19 +397,25 @@ def refresh(username = nil)
         end
 
         say("Refreshed #{processed} accounts#{dry_run}", :green, true)
-      elsif username.present?
-        username, domain = username.split('@')
-        account = Account.find_remote(username, domain)
-
-        if account.nil?
-          say('No such account', :red)
-          exit(1)
-        end
+      elsif usernames.size > 0
```

Comment:
Sorry I'm making you change that again, but the linter actually has a good suggestion:
```suggestion
      elsif !usernames.empty?
```